scaling_performance/optimizing_storage.adoc

```diff
@@ -127,17 +275,16 @@ Overlay is also supported for Docker use cases as of Red Hat Enterprise Linux
 potentially improve density by reducing overall memory utilization.
 
 [[benefits-of-using-the-overlay-graph-driver]]
-== Benefits of Using the Overlay Graph Driver with SELinux
+=== Benefits of Using the Overlay Graph Driver with SELinux
 
-The default Docker storage configuration on Red Hat Enterprise Linux (RHEL)
-continues to be Device Mapper. While the use of Overlay as the containers
-storage back-end is being monitored, moving RHEL to Overlay as the default in
-future releases is under consideration. As of RHEL 7.2, Overlay became a
-supported graph driver. As of RHEL 7.4, SELinux and the Overlay2 graph driver
+The default Docker storage configuration on Red Hat Enterprise Linux continues to be device mapper. While the use of overlay as the containers
```

Comment:
Is containers supposed to be possessive here?

Reply:
Yes.